packages/larva-patterns/modules/vlanding-playlist/vlanding-playlist.prototype.js

```diff
@@ -0,0 +1,34 @@
+const clonedeep = require( 'lodash.clonedeep' );
+
+const o_header = clonedeep ( require( '../../objects/o-header/o-header.prototype' ) );
+const o_card = clonedeep ( require( '../../objects/o-card/o-card.prototype' ) );
```

Comment:
No spaces after function calls - soon will have a lint-fix command in here.

Reply:
Good catch, That was by mistake.